pydarn/io/borealis/borealis_convert.py

```diff
@@ -397,11 +412,12 @@ def _convert_bfiq_to_iqdat(self):
         """
         try:
             recs = []
-            slice_id = int(os.path.basename(self.borealis_filename).split('.')[4])
             for record in self.borealis_records.items():
-                    record_dict = self.__convert_bfiq_record(slice_id, record, 
-                                                             self.borealis_filename)
-                    recs.append(record_dict)
+                record_dict = self.__convert_bfiq_record(
+                                    self.borealis_slice_id,
+                                    record, 
+                                    self.borealis_filename)
+                recs.append(record_dict)
             dmap_recs = dict2dmap(recs)
         except Exception as e:
             raise borealis_exceptions.BorealisConvert2IqdatError(e)
```

Comment:
Did you purposely re-raise the exception to get rid of the traceback? 
Maybe use `raise from` syntax to keep the traceback. 

I think we need to discuss this re-raising of exceptions because it is getting confusing.

Reply:
changed to raise from